### PR TITLE
add test module to use e2e tests

### PR DESCRIPTION
**Pull request by EfimovVladimir**

### CODE REVIEW

**Comment by scottdraves** [comment]
can you set this up to use yarn instead of npm?

**Comment by EfimovVladimir** [comment]
Changed to use yarn instead npm.
Changed to use Webdriver.io instead Protractor.

File: test/runner.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import os
+import sys
+import subprocess
+import signal
+
+here = os.path.abspath(os.path.dirname(__file__))
+beakerx_dir = os.path.abspath(os.path.join(here, ".."))
+test_dir = here
+
+# update environment
```
Comment (LeeTZ) [inline test/runner.py]: We should get rid of Windows line end char '^M' here

File: test/tests/beakerx.po.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+var BeakerXPageObject = function() {
+
+    this.baseURL = 'http://127.0.0.1:8888/tree/demoFiles';
```
Comment (LeeTZ) [inline test/tests/beakerx.po.js]: *.js files should have two-space indentation.

File: test/wdio.conf.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
+exports.config = {
```
Comment (LeeTZ) [inline test/wdio.conf.js:17]: should have a copyright banner here.

**Comment by LeeTZ** [comment]
For the integration test It works well!

**Pull request merged**